content/doc/book/system-administration/reverse-proxy-configuration-nginx.adoc

```diff
@@ -49,6 +49,10 @@ server {
   access_log      /var/log/nginx/jenkins/access.log;
   error_log       /var/log/nginx/jenkins/error.log;
   
+  # If Nginx fail with 'No such file or directory' replace the two previous lines
```

Comment:
I feel like adding more "cases" makes the resulting page really hard to read.

WDYT about simply updating the original 2 lines with your configuration? That would be simpler and should solve the issue you had?

Reply:
I kinda feel the same,  but for a reason that they're both valid and the previous two lines can also work in some environments.
Giving this a second thought, since the original config may not work for all, I think it would be best as you stated to just update original lines with what's valid for my case and also would still work for others that tried the original config with no issue